src/tests/HDFFile.cxx

```diff
@@ -478,6 +478,14 @@ class T_CommandHandler : public testing::Test {
     }
 
     H5Fclose(fid);
+
+    {
+      // now try to recreate the file for testing:
+      auto x =
+          H5Fcreate("tmp-ev42.h5", H5F_ACC_TRUNC, H5P_DEFAULT, H5P_DEFAULT);
+      ASSERT_GE(x, 0);
+      H5Fclose(x);
+    }
```

Comment:
If you've given it its own scope and a title, why not make it a method? `data_ev42()` is already far too large and difficult to follow.
```
void recreate_file_for_testing() {
  auto x =
    H5Fcreate("tmp-ev42.h5", H5F_ACC_TRUNC, H5P_DEFAULT, H5P_DEFAULT);
  ASSERT_GE(x, 0);
  H5Fclose(x);
}
```